### PR TITLE
Remove Library

### PR DESCRIPTION
**Pull request by 0x657373656C6C**
Removed https://github.com/yourapiexpert/ATCommands as the repository username has changed and the library is old and defunct.

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @0x657373656C6C. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/06/20 15:57:00 Loaded 7079 libraries from DB
Removing ATCommands
Success!
```

**Pull request merged**
